frontend/src/metabase/nav/hooks.ts

```diff
@@ -23,7 +23,5 @@ export function useShouldShowDatabasePromptBanner(): boolean | undefined {
     return undefined;
   }
 
-  const onlyHasSampleDatabase =
-    databases.length === 1 && databases[0].is_sample;
-  return onlyHasSampleDatabase;
+  return databases.every(database => database.is_sample);
```

Comment:
There was a [discussion here](https://metaboat.slack.com/archives/C05EWA11Z1V/p1689763291003789?thread_ts=1689762843.462409&cid=C05EWA11Z1V). Basically, the previous condition isn't correct because you can remove the only sample database and it will also remove the database prompt banner when that should stay unless users add their own database.